ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/DiskPolicy.hs

```diff
@@ -12,10 +12,11 @@ module Ouroboros.Consensus.Storage.LedgerDB.DiskPolicy (
 import           Data.Time.Clock (secondsToDiffTime)
 import           Data.Word
 import           NoThunks.Class (NoThunks, OnlyCheckWhnf (..))
-
+import           Cardano.Slotting.Slot (SlotNo)
```

Comment:
The blank line was intentional: see "Import lists:" in https://github.com/input-output-hk/ouroboros-network/blob/master/ouroboros-consensus/docs/StyleGuide.md

`Control.Monad.Class.MonadTime` is (surprisingly) in the network layer's `io-sim-classes`.

It should be:

```
import NoThunks.Class (NoThunks, OnlyCheckWhnf (..))

import Cardano.Slotting.Slot (SlotNo, WithOrigin)

import Control.Monad.Class.MonadTime
```